src/libs/ppscale/ppscale.c

```diff
@@ -24,21 +24,15 @@ int pp_getscale /* calculate integer scales for pixel-perfect magnification */
 	double parweight,      /* weight of PAR in scale estimation */
 	int    *sx,  int *sy   /* horisontal and vertical scales    */
 ) /* returns -1 on error and 0 on success */
-{	int    sxc = 0, syc = 0, sxm = 0, sym = 0;   /* current and maximum x and y scales     */
-	int    exactpar = 0;             /* whether to enforce exact aspect ratio  */
-	double parrat = 0;               /* ratio of current PAR and target PAR    */
-	double errpar = 0, errsize = 0, err = 0; /* PAR error, size error, and total error */
-	double errmin = 0;               /* minimal error so far                   */ 
-	double parnorm = 0;              /* target PAR "normalised" to exceed 1.0  */
-	double srat = 0;                 /* ratio of maximum size to current       */
-
-	if /* check for invalid inputs: */
-	(	win <= 0    || hin <= 0    ||
+{
+	if /* sanity checks: */
+	(
+		win <= 0    || hin <= 0    ||
 		win >  wout || hin >  hout ||
 		par <= 0.0  || parweight <= 0 ||
 		sx == NULL || sy == NULL
-	)
-	return -1;
+		)
+		return -1;
 
```

Comment:
indentation above looks off

Reply:
Thanks - fixed that up.

Also noticed I wasn't adhering to @ant-222's original spacing around some of the new `if/else` checks, so added that in the last commit.  Hopefully maintains the original style now.

Tested and still working.